File: backend/internal/models/pricing.go
package models

type PricingConfig struct {
	GermanCities []CityPricing   `json:"german_cities"`
	Materials    []MaterialPrice `json:"materials"`
}

type CityPricing struct {
	ID            string `json:"-"`
	CityName      string `json:"name"`
	PricePerTooth int    `json:"price_eur_per_tooth"`
	SortOrder     int    `json:"-"`
}

type MaterialPrice struct {
	ID            string `json:"-"`
	Key           string `json:"key"`
	NameDE        string `json:"name_de"`
	NameEN        string `json:"name_en"`
	PricePerTooth int    `json:"price_eur_per_tooth"`
	SortOrder     int    `json:"-"`
}

File: frontend/src/types/pricing.ts
export interface CityPricing {
  name: string;
  price_eur_per_tooth: number;
}

export interface MaterialPrice {
  key: string;
  name_de: string;
  name_en: string;
  price_eur_per_tooth: number;
}

export interface PricingConfig {
  german_cities: CityPricing[];
  materials: MaterialPrice[];
}
